service/test/controllers/organisationmanagement/OrganisationControllerTest.java

```diff
@@ -1,404 +1,163 @@
-    innerMap.put(JsonKey.STATUS, new BigInteger("1"));
-    requestMap.put(JsonKey.REQUEST, innerMap);
-    String data = mapToJson(requestMap);
-
-    JsonNode json = Json.parse(data);
-    RequestBuilder req =
-        new RequestBuilder().bodyJson(json).uri("/v1/org/status/update").method("POST");
-    req.headers(headerMap);
-    Result result = route(req);
-    assertEquals(404, result.status());
+  public void testUpdateOrgStatusSuccess() {
+    Result result =
+        performTest(
+            "/v1/org/status/update",
+            "PATCH",
+            createOrganisationRequest(null, orgId, false, null, status, null));
+    assertEquals(getResponseCode(result), ResponseCode.success.getErrorCode().toLowerCase());
+    assertTrue(getResponseStatus(result) == 200);
   }
 
   @Test
-  public void testgetOrgDetails() {
-    PowerMockito.mockStatic(RequestInterceptor.class);
-    when(RequestInterceptor.verifyRequestData(Mockito.anyObject()))
-        .thenReturn("{userId} uuiuhcf784508 8y8c79-fhh");
-    Map<String, Object> requestMap = new HashMap<>();
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.ORGANISATION_ID, "org123");
-    innerMap.put(JsonKey.STATUS, new BigInteger("1"));
-    requestMap.put(JsonKey.REQUEST, innerMap);
-    String data = mapToJson(requestMap);
-
-    JsonNode json = Json.parse(data);
-    RequestBuilder req = new RequestBuilder().bodyJson(json).uri("/v1/org/read").method("POST");
-    req.headers(headerMap);
-    Result result = route(req);
-    assertEquals(200, result.status());
+  public void testUpdateOrgStatusFailureWithoutOrgId() {
+    Result result =
+        performTest(
+            "/v1/org/status/update",
+            "PATCH",
+            createOrganisationRequest(null, null, false, null, status, null));
+    assertEquals(getResponseCode(result), ResponseCode.mandatoryParamsMissing.getErrorCode());
+    assertTrue(getResponseStatus(result) == 400);
   }
 
   @Test
-  public void testaddMemberToOrganisation() {
-    PowerMockito.mockStatic(RequestInterceptor.class);
-    when(RequestInterceptor.verifyRequestData(Mockito.anyObject()))
-        .thenReturn("{userId} uuiuhcf784508 8y8c79-fhh");
-    Map<String, Object> requestMap = new HashMap<>();
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.ORGANISATION_ID, "org123");
-    innerMap.put(JsonKey.USER_ID, "userId");
-    requestMap.put(JsonKey.REQUEST, innerMap);
-    String data = mapToJson(requestMap);
-
-    JsonNode json = Json.parse(data);
-    RequestBuilder req =
-        new RequestBuilder().bodyJson(json).uri("/v1/org/member/add").method("POST");
-    req.headers(headerMap);
-    Result result = route(req);
-    assertEquals(200, result.status());
+  public void testGetOrgDetailsSuccess() {
+    Result result =
+        performTest(
+            "/v1/org/read",
+            "POST",
+            createOrganisationRequest(null, orgId, false, null, status, null));
```

Comment:
Why get org is using createOrganisationRequest? This should be separate if at all.